pkg/parser/pipelineyml/visitor_cron.go

```diff
@@ -120,11 +120,6 @@ func (v *CronVisitor) Visit(s *Spec) {
 		v.nextTimes = append(v.nextTimes, nextTime)
 		scheduleFrom = nextTime
 	}
-
-	// compensator
-	if s.CronCompensator == nil {
-		s.CronCompensator = &DefaultCronCompensator
-	}
```

Comment:
Can you delete yaml part already with default con compensator?
Users don't care about this. And they can get polished default compensator configs if we adjusted.